src/Tasks/InstallWindowsScheduledTask/task.json

```diff
@@ -28,393 +28,403 @@
+			"groupName": "ScheduledTaskSettings"
+		},
+		{
+			"name": "ScheduleStartTimeRandomDelayInMinutes",
+			"type": "string",
+			"label": "Start Time Random Delay (in Minutes)",
+			"visibleRule": "ScheduledTaskDefinitionSource = Inline",
+			"required": false,
+			"defaultValue": "0",
+			"helpMarkDown": "The maximum amount of minutes to delay the start time of the trigger. The delay will be a random value between zero and the number of minutes specified here.\u003cbr /\u003e\u003cbr /\u003eYou may want to use this to stagger when the job starts if you have many jobs set to start at the same time.",
+			"groupName": "ScheduledTaskSettings"
+		},
+		{
+			"name": "ScheduledTaskAccountToRunAsOptions",
+			"type": "radio",
+			"label": "User To Run As",
+			"required": true,
+			"defaultValue": "System",
+			"helpMarkDown": "The user that the Scheduled Task should run as.",
+			"options": {
+				"System": "System",
+				"LocalService": "Local Service",
+				"NetworkService": "Network Service",
+				"CustomAccount": "Custom User or Service Account"
+			},
+			"groupName": "ScheduledTaskSettings"
+		},
+		{
+			"name": "CustomAccountToRunScheduledTaskAsUsername",
+			"type": "string",
+			"label": "Scheduled Task Username",
+			"visibleRule": "ScheduledTaskAccountToRunAsOptions = CustomAccount",
+			"required": true,
+			"defaultValue": "",
+			"helpMarkDown": "The name of the user or system account that the Scheduled Task should run as.",
+			"groupName": "ScheduledTaskSettings"
+		},
+		{
+			"name": "CustomAccountToRunScheduledTaskAsPassword",
+			"type": "string",
+			"label": "Scheduled Task Password",
+			"visibleRule": "ScheduledTaskAccountToRunAsOptions = CustomAccount",
+			"required": false,
+			"defaultValue": "",
+			"helpMarkDown": "The password of the user or system account that the Scheduled Task should run as.\u003cbr /\u003e\u003cbr /\u003eAs a best practice, a secret variable should be passed in here so that other users cannot see the password in plain text.",
+			"groupName": "ScheduledTaskSettings"
+		},
+		{
+			"name": "ShouldScheduledTaskBeEnabledString",
+			"type": "boolean",
+			"label": "Enabled",
+			"visibleRule": "ScheduledTaskDefinitionSource = Inline",
+			"required": true,
+			"defaultValue": "true",
+			"helpMarkDown": "Select this to have the Scheduled Task be enabled after it is installed; Unselect this to have it be installed as disabled.",
+			"groupName": "ScheduledTaskSettings"
+		},
+		{
+			"name": "StartWhenAvailableString",
+			"type": "boolean",
+			"label": "Start Task When Available",
+			"visibleRule": "ScheduledTaskDefinitionSource = Inline",
+			"required": true,
+			"defaultValue": "false",
+			"helpMarkDown": "Indicates that Task Scheduler can start the task at any time after its scheduled time has passed.",
```

Comment:
Awesome, I'm glad that you copied this verbatim [from the MS docs](https://docs.microsoft.com/en-us/powershell/module/scheduledtasks/new-scheduledtasksettingsset?view=windowsserver2022-ps) 👍